src/components/upsell/joinChannel.js

```diff
@@ -92,13 +95,31 @@ class JoinChannel extends React.Component<Props, State> {
       });
   };
 
+  hideUpsell = () => {
+    this.setState({
+      isOpen: false,
+    });
+  };
+
   render() {
-    const { isLoading } = this.state;
+    const { isLoading, isOpen } = this.state;
     const { channel, community, currentUser } = this.props;
 
+    if (!isOpen) {
+      return (
+        <FloatingButton
+          loading={isLoading}
+          onClick={!currentUser ? this.login : this.toggleSubscription}
+          icon={!currentUser ? 'door-enter' : 'plus'}
```

Comment:
Maybe we should add the `door-enter` icon to the `Login or sign up` button as well now.